src/components/RadialSlider/RadialSlider.tsx

```diff
@@ -61,6 +64,22 @@ const RadialSlider = (props: RadialSliderProps & typeof defaultProps) => {
     centerValue,
   } = useRadialSlider(props);
 
+  useEffect(() => {
+    //check max value length
+    const maxLength = ('' + max).length;
```

Comment:
@parth-simform Can use please convert that max to string as follows ?
`const maxLength = max?.toString()?.length`

Reply:
Resolved